Jenkinsfile

```diff
@@ -8,11 +8,8 @@ pipeline {
         timestamps()
     }
     parameters {
-        booleanParam(name: 'unit_validate', defaultValue: true, description: 'amd64 (x86_64) unit tests and vendor check')
+        booleanParam(name: 'validate', defaultValue: true, description: 'amd64 (x86_64) validate and vendor check')
```

Comment:
Can we cover this on GHA?

Reply:
Yes I have a branch for these in a follow-up, didn't want to add too much in https://github.com/moby/moby/pull/44034